extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -268,16 +267,18 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
         DicomMetadataStore.studyLoaded(study);
       }
 
-      // Google Cloud Healthcare doesn't return StudyInstanceUID, so we need to add
-      // it manually here
-      seriesSummaryMetadata.forEach(aSeries => { aSeries.StudyInstanceUID = StudyInstanceUID })
-
       DicomMetadataStore.addSeriesMetadata(seriesSummaryMetadata, madeInClient);
 
-      const numberOfSeries = seriesPromises.length;
-      seriesPromises.forEach(async (seriesPromise, index) => {
-        const instances = await seriesPromise;
-        storeInstances(instances);
+      const numberOfSeries = seriesSummaryMetadata.length;
+      seriesSummaryMetadata.forEach(async (series, index) => {
```

Comment:
Is this truly an async operation? Do we have to wait for all of the instances to be loaded before we display the series?

Reply:
Yes